DesignSafe-CI/portal
# DES-2047 and 2125

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-2047](https://jira.tacc.utexas.edu/browse/DES-2047)
* [DES-2125](https://jira.tacc.utexas.edu/browse/DES-2125)

## Summary of Changes: ##
All links to the Curation Office hours should go to `/facilities/virtual-office-hours/`now.
Location of the Edit Project link has been changed.

## Testing Steps: ##
To test DES-2047:
1. Go to any project and check that the Edit Project link is left aligned (in the same way as the Download Dataset button is on published projects.  Edit Project is just a link and not a button, but it should be in the same place.

To test DES-2125:
1. I'm not sure there's a best way to test this one, but every location where it mentions Office Hours should have the link to the correct place.  This may or may not work locally (I don't have that path locally), but the url preview at the bottom left of the web browser should show the correct path.

## UI Photos:

## Notes: ##
